hr_holidays_settings/views/hr_holidays_config_settings.xml

```diff
@@ -32,4 +32,9 @@
               action="hr_holidays_config_settings_action"
               groups="base.group_system"/>
 
+    <menuitem id="menu_hr_holidays_status" name="Leave Types"
+          parent="hr_holidays.menu_hr_holidays_configuration" sequence="10"
+          action="open_view_holiday_status"
```

Comment:
you need the complete external ID here I think: `hr_holiday.open_view_holiday_status` I guess but check it please